src/modules/overview/actions/fetchItemsByFilter.ts

```diff
@@ -5,7 +5,7 @@ export enum ItemsChannel {
   all = 'All',
   fineArts = 'FineArts',
   sports = 'Sports',
-  comics = 'Conicbooks',
```

Comment:
Are you sure? Was that typo error fixed on backend?
This is the request parameter for API.

Reply:
Is it typo on our side or on the backend?

Reply:
In fact we need keep the parameters the same in both creat method and query method. But document isn't the sam. Confirmed with the guy to use the Comicbooks